README.md

```diff
@@ -22,8 +22,8 @@ rm build/*.* && yarn build
 ```
 createdb boxoffice
 pip install -r requirements.txt
-./manage.py dbconfig | sudo -c postgres psql boxoffice
-./manage.py createdb
+flask dbconfig | sudo -c postgres psql boxoffice
+flask createdb
```

Comment:
This does not exist right now. Is it coming from coaster?

Reply:
It's not required for the tests. They do createdb in setup.